spec/support/request_helpers.rb

```diff
@@ -561,4 +561,64 @@ def stub_course
     stub_timeline
     stub_users
   end
+
+  def stub_lift_wing_response
```

Comment:
The idea with all the stub methods in `request_helpers` is to use `stub_request` to replace real web requests to specific domains or URLs, not to directly return data.

Reply:
I understood what the stubbing required,  but the data returned from these api calls are large, so i did not want to fill the entire file with those data,

Reply:
I smaller response is fine, but anything in this file should still rely on `stub_request` to do its work.

Reply:
Alright, I'll update that and push